src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -917,4 +917,64 @@
       it { expect(package.attribs.where(attrib_type: attribute_type)).to be_empty }
     end
   end
+
+  describe 'POST #new_release_request' do
+    let(:project) { create(:maintenance_incident_project) }
+
+    before do
+      login user
+    end
+
+    context 'with skiprequest param' do
+      before do
+        post :new_release_request, params: { project: project, skiprequest: true }
+      end
+
+      it { expect(response).to redirect_to( project_show_path(project)) }
+    end
+
+    context 'when raises an APIException' do
```

Comment:
I just forgot to add this line:

`allow_any_instance_of(BsRequest).to receive(:save!).and_raise(APIException)`